tests/cleaner_tests/existing_archive.py

```diff
@@ -96,8 +96,8 @@ def test_no_empty_obfuscations(self):
             map_json = json.load(mf)
         for mapping in map_json:
             for key, val in map_json[mapping].items():
-                assert key, "Empty key found in %s" % mapping
-                assert val, "%s mapping for '%s' empty" % (mapping, key)
+                assert key, f"Empty key found in {ARCHIVE}"
```

Comment:
Shouldn't it be `{mapping}` instead of `{ARCHIVE}`? Or why the change?

Reply:
That was simply a mistake :) Fixed now.